<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorWidget</class>
 <widget class="QWidget" name="MonitorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">
QWidget#widget,#MonitorWidet
{
	border-image:none;
}

QWidget#widget_2
{
	border-image: url(:/images/images/background_2.jpg);
}

QTabWidget
{
	
	
	border-color: rgb(255, 255, 255);
	border-bottom-color: rgb(255, 255, 127);
}

QTabBar::tab{ 
	border-image:none;
	color: white; 
	padding: 9px;
	border: 0px solid transparent; 
	border-image: url(:/images/images/back_title.png);
} 

QTabBar::tab:selected {	
	border-image: url(:/images/images/list_left_click.png);
	border-radius:6px;
 	white:red;
	font-size:12px;
	font-weight:bold;
}

QTabBar::tab:!selected {
	margin-top: 5px;
	border-image: url(:/images/images/list_left_2.png);
	border-radius:4px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_list" native="true">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_show" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTabWidget" name="tabWidget">
              <property name="tabShape">
               <enum>QTabWidget::Rounded</enum>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="elideMode">
               <enum>Qt::ElideMiddle</enum>
              </property>
              <property name="usesScrollButtons">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="tab_1">
               <attribute name="title">
                <string>Page_1</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>Page_2</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
